rust_src/src/buffers.rs

```diff
@@ -16,6 +16,11 @@ pub type LispBufferRef = ExternalPtr<Lisp_Buffer>;
 
 impl LispBufferRef {
     #[inline]
+    pub fn zv(self) -> ptrdiff_t {
```

Comment:
should be `&self` too for consistency (although it doesn't matter since our refs are just pointers).